packages/realm-web/src/services/HTTPService.ts

```diff
@@ -53,7 +57,7 @@ class HTTPService implements Realm.Services.HTTP {
      * @param options Options related to the request.
      * @returns The response.
      */
-    get(url: string, options: Realm.Services.HTTP.RequestOptions = {}) {
+    get(url: string, options: RequestOptions = {}): Promise<Response> {
```

Comment:
can we declare these as async and use async/await

Reply:
No need - an async function still needs the `Promise<...>` type annotation if the type can't be inferred from the return statement.